FirstProject/new-evaluation.html

```diff
@@ -517,11 +524,13 @@ <h2 class="description_section" style="color: #03A9F4;">Workflow, Leadership & S
                     </li>
                 </ul>
             </fieldset> <br>
-            <input class="submit_button" type="submit" value="SUBMIT" style="float: right; margin-top: 30px; padding: 15px 40px; margin-right: 100px;">
+            <div class="#" style="width: 73%; margin: 0 auto;">
+                <input class="submit_button" type="submit" value="SUBMIT" style="float: right; margin-top: 30px; padding: 15px 40px; margin-right: 100px;">
+            </div>
         </form>
     </div>
 
-    <footer class="container-fluid footer_side" style="margin-top: 80px;">
+    <footer class="container-fluid" style="margin-top: 80px;">
         <p class="footer_area">Copyright@Sofvision 2017</p>
```

Comment:
I use to miss letters especially when I text messages but I will fix it for sure. :)